python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -130,6 +131,18 @@ class Actor extends React.Component<Props & WithStyles<typeof styles>, State> {
             }
           ];
 
+    // Construct the custom message from the actor.
+    let actorCustomDisplay: JSX.Element[] = []
+    if (actor.state !== -1 && actor.webuiDisplay) {
+      Object.keys(actor.webuiDisplay).sort().forEach(key =>{
+        actorCustomDisplay.push(<Typography className={classes.webuiDisplay}>
+          &nbsp; &nbsp; {key}: {actor.webuiDisplay![key]}
+          </Typography>)
+        actorCustomDisplay.push(<Divider></Divider>)
```

Comment:
A full-width divider between every key-value pair might be a little heavy, can we drop this and the corresponding import? If so, we could also turn this into a `map` instead of a `forEach`.